osquery/core/init.cpp

```diff
@@ -603,7 +603,7 @@ void Initializer::start() const {
     if (tool_ == ToolType::DAEMON) {
       LOG(WARNING) << message;
     } else {
-      LOG(INFO) << message;
```

Comment:
Ah, I think this behavior is a recently-fixed regression. See: https://github.com/facebook/osquery/commit/976db066c05ee9cdf650c056b641ba5f588185ee